Source/src/UI/WindowProject.cpp

```diff
@@ -162,77 +170,80 @@ void Hachiko::WindowProject::ShowDir(PathNode& node)
 
 void Hachiko::WindowProject::ShowFilesOnFolder()
 {
-    ImGui::BeginChild("main_col");
-
-    const float w = ImGui::GetContentRegionAvail().x;
-    int columns = static_cast<int>(w) / static_cast<int>(96 * 1.f);
-    columns = std::max(columns, 1);
-    int tile_count = all_assets.children.empty() ? all_assets.children.size() : 50;
-    const int row_count = 1;
-
-    auto callbacks = [this](PathNode& node, int idx) {
-        if (ImGui::IsItemHovered())
-            ImGui::SetTooltip("%s", node.path.c_str());
-        if (ImGui::BeginDragDropSource(ImGuiDragDropFlags_SourceAllowNullID))
-        {
-            ImGui::Text("%s", node.path.c_str());
-            ImGui::SetDragDropPayload("path", node.path.c_str(), strlen(node.path.c_str()) + 1, ImGuiCond_Once);
-            ImGui::EndDragDropSource();
-        }
-        else if (ImGui::IsItemHovered())
-        {
-            if (ImGui::IsMouseReleased(0))
-            {
-                // selected_resource = find on list resource module resource
-            }
-            else if (ImGui::IsMouseReleased(1))
+    if (ImGui::BeginChild("main_col"))
+    {
+        const float w = ImGui::GetContentRegionAvail().x;
+        int columns = static_cast<int>(w) / static_cast<int>(96 * 1.f);
+        columns = std::max(columns, 1);
+        int tile_count = all_assets.children.empty() ? all_assets.children.size() : 50;
```

Comment:
@Bernatmago this `tile_count` logic seems kinda weird? Should it be other way around? Or doing the check of `all_assets.children.empty()` and then if its `true` we don't need to check for `all_assets.children.size()` since it's gonna expand to `0`.

Reply:
I don't like those magic numbers there, but this is pretty much the original logic, just adding the login inside the child node, reviewing that logic is out of the scope of this PR